### DISCUSSION TITLE
Has relative path support been added to the Modelstore (since 0.74)?

### DISCUSSION THREAD

**Discussion by shsnyder** [operatorai/modelstore#219] [General] [answered]
I have a project that uses modelstore==0.74.
In the versions file for each of the models the absolute path to the modelstore is used to locate models to be loaded (modelstore.load() ). This path is the one where the modelstore was created. 
If the modelstore directory tree is moved to a different directory, model loads will fail.

Has the ability to specify a modelstore root directory been added to the load process so that the modelstore csn be "rooted" in a different directory?

**Comment by nlathia** [comment, marked as answer]
Hello! I added this into 0.0.77; you can see the full change log here:

https://github.com/operatorai/modelstore/blob/main/CHANGELOG.md

Please feel free to open an issue if this doesn't work for you. I suspect it won't easily work for models that were saved with an earlier version of the library.
